data/mods/gen9ssb/abilities.ts

```diff
@@ -699,6 +699,34 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
 		flags: {},
 	},
 
+	// Lunell
+	lowtidehightide: {
+		shortDesc: "Switch-in sets Gravity, water immune, traps water type foes.",
+		name: "Low Tide, High Tide",
```

Comment:
I'm unsure if commas are allowed in ability/item/move names, its a special character.